index.jade

```diff
@@ -0,0 +1,38 @@
+<?php include('connect.php'); ?>
+doctype
+html
+  head
+    title Make Your Pizza
+  body
+    .container
+      h1 Choose Your Toppings
+      form(action="review.php" method="POST")
+        .group
+          label(for="cheese") Cheese
+          <?php while($row = mysqli_fetch_assoc($results['cheeses'])) { ?>
```

Comment:
Ok, so this is why your `<?php` line is ending up on the same line as the label or whatever line is before it.  In jade if you want to add plain text (which is what php is in your html file even though it gets processed by the server) you need to add a `|` to the beginning of the line, example:

```jade
| <?php while($row = mysqli_fetch_assoc($results['cheeses'])) { ?>
```